canonical/snapcraft.io
# WD-3730 - Remove brandstore configs

## Done
Remove `webapp/configs` folder

## How to QA
- Visit the [demo](https://snapcraft-io-4265.demos.haus/) - the site should load

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-3730
